modules/dreamview/frontend/src/components/TeleopMonitor/CarTeleop.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { inject, observer } from "mobx-react";
+
+import { TELEOP_WS } from "store/websocket";
+
+import CheckboxItem from "components/common/CheckboxItem";
+import MonitorSection from "components/TeleopMonitor/MonitorSection";
+
+import itemIcon from "assets/images/icons/teleop_item.png";
+
+
+@inject("store") @observer
+export default class CarTeleOp extends React.Component {
+    componentDidMount() {
+        TELEOP_WS.initialize();
+    }
+
+    componentWillUnmount() {
+        TELEOP_WS.close();
+    }
+
+    render() {
+        const { teleop } = this.props.store;
+
+        return (
+            <div className="monitor teleop">
+                <div className="monitor-header">
+                    <div className="title">Car Teleop Controls</div>
+                </div>
+
+                <div className="monitor-content">
+                    <MonitorSection title="Audio" icon={itemIcon}>
+                        <div className="section-content-row" key={name}>
+                            <CheckboxItem id='teleopAudio'
```

Comment:
do we have mic mute button for car teleop?

Reply:
I think it would be nice to have one